### PR TITLE
Clean up unused old federation code in kubetest

### PR DESCRIPTION
**Pull request by krzyzacy**
cleaning up old federation v1 code in kubetest which is not used for a while now
/assign @fejta @BenTheElder 
cc @shashidharatd

### CODE REVIEW

File: kubetest/build.go [hunk 1]
```diff
@@ -94,37 +94,3 @@ func (b *buildStrategy) Build() error {
...
-	buildStrategy
-}
-
-func (b *buildFederationStrategy) Type() string {
-	return "buildFederationStrategy"
-}
-
-func (b *buildIngressGCEStrategy) Type() string {
-	return "buildIngressGCEStrategy"
-}
```
Comment (krzyzacy) [inline kubetest/build.go]: also nothing references this buildIngressGCEStrategy today

**Comment by krzyzacy** [comment]
`fatal: unable to access 'https://go.googlesource.com/lint/': The requested URL returned error: 502` lol

File: kubetest/anywhere.go [hunk 2]
```diff
@@ -317,6 +317,65 @@ func (_ *kubernetesAnywhere) KubectlCommand() (*exec.Cmd, error) { return nil, n
...
+}
+
+func (m *multiClusterDeployment) Type() string {
+	return "multiClusterDeployment"
+}
+
+func (m *multiClusterDeployment) Enabled() bool {
+	return len(m.clusters) > 0
+}
+
```
Comment (krzyzacy) [inline kubetest/anywhere.go]: cc @neolit123 I dumped this struct from federation.go into k-a.go, although seems there's no CI job uses this, let me know if you prefer just get rid of that :-)

Comment (neolit123) [reply kubetest/anywhere.go]: @krzyzacy 
+1 for removing these multicluster bits.
i'm hoping that we can remove `anywhere.go` as a whole in a couple of releases completely.

Comment (krzyzacy) [reply kubetest/anywhere.go]: thanks! cleaned up :-)

**Comment by neolit123** [comment]
this is good to go on the `anywhere*.go` side.

**Pull request merged**